File: judge/src/tested/languages/java/templates/evaluator_executor.java
## Responsible for generating a function expression to a custom evaluator.
<%page args="evaluator,function" />
import java.util.*;

public class EvaluatorExecutor {

    public static void main(String[] args) throws Exception {
        var result = <%include file="function.mako" args="function=function" />;
        Values.sendEvaluated(result);
    }
}

File: judge/src/tested/languages/kotlin/templates/EvaluatorExecutor.kt
## Responsible for generating a function expression to a custom evaluator.
<%page args="evaluator,function" />\

import java.io.PrintWriter

fun main() {
    val result = <%include file="function.mako" args="function=function" />
    val writer = PrintWriter(System.out)
    sendEvaluated(writer, result)
    writer.flush()
}
